### ISSUE TITLE
Alternativa a raspberry 3

### ISSUE THREAD

**Issue by Italyguru** [geekcookies/geektalks#5]
E' presto per parlare, prima di averla vista all'opera, ma la più recente incarnazione del lampone mi ha lasciato un po' deluso. L'ho trovata inferiore alla concorrenza.
Sto cercando un'altra scheda per sostituire il mio raspberry 1B, più potente, dal prezzo accessibile, per attività varie ma che comprendono un sistema operativo (no Arduino o simili. Può doversi interfacciarsi a sensori come l'ESP)
Pensavo alla Pine 64 o alla UDOO, ne consigliate qualcuna?

**Comment by kidpixo** [comment]
Concordo pienamente, mi pare tanto un classico upgrade incrementale, come si sono visti un tutte le tecnologie quando si arrivati alla stagnazione (PC, laptop, Tablet, smartphone). Tra l'altro sa tanto di risposta al pine64... Un altra che mi ha consigliato Giuliano é ODROID-C2, a 64-bit con uno slot eMMC5.0 HS400 Flash Storage (mi pare li faccia Toshiba?). Comunque consulta in rete per vedere cosa soddisfa le tue esigente tipo http://www.board-db.org/.

**Comment by cesco78** [comment]
Io l'ho ordinato, sono curioso e avere un po' di velocità in più, la WiFi e il Bluetooth già integrati allo stesso presso di prima non mi pare una scelta così brutta.
Le alternative sono valide, Udoo è molto interessante, ma il prezzo?

**Comment by Italyguru** [comment]
Io invece mi sono quasi convinto a provare la pine64... certo, tutti amiamo il lampone, e il prezzo è un grande incentivo, resta il fatto che mi pare che l'azienda abbia fatto una mossa molto (troppo?) rapida e poco impattante. Maquillage tecnico più che innovazione.

**Comment by cesco78** [comment]
Provare nuovi dispositivi è sempre una gran cosa, come feci io al tempo della Udoo :) facci sapere come va!

**Comment by gpz500** [comment]
Io proverei volentieri la Udoo x86 (tra l'altro è una azienda in parte italiana, spin-off dell'Università di Siena, mi pare) che dovrebbe uscire a novembre, adesso in fase fund raising su kickstarter. Ma mi sembra un po' cara: parte da $88.

**Comment by kidpixo** [comment]
Concordo, peró per quel che fá vale il costo direi, ti ci fai un computer vero volendo. 
Per soluzioni embedded fini a se stesse (IoT innanzitutto, ma anche un fileserver/NAS e robe simili) é davvero troppo costosa.

**Comment by cesco78** [comment]
Il problema alla fine è solo uno: provare tutto il mondo ogni volta che esce qualcosa di nuovo o sceglierne una e cercare di sfruttare a fondo quella? Sono passato dalla fase 1 (esce, lo compro, provo, resta a prendere la polvere) alla fase 2 (mi piace il raspberry, uso quello e basta).

Poi, continuo a pensare l'utilizzo di Kikstarter da parte di una azienda già lanciata, che quindi ribalta il rischio di impresa sugli acquirenti (come ha già fatto Pebble per il Time) sia una cosa eticamente sbagliata

**Comment by gpz500** [comment]
@cesco78: concordo perfettamente su quello che dici riguardo a Kickstarter per aziende già avviate.

Sul Raspberry Pi 3: ce l'ho da un po' di tempo e... mi hanno deluso le prestazioni del Wi-Fi: il link è molto instabile e variabile. Ho anche un Raspberry Pi B con dongle Wi-Fi USB che va molto meglio. Ma magari è un problema del mio esemplare... a voi come va?

**Comment by kidpixo** [comment]
Bé dipende un pó dalle dimensioni dell'azienda: per la Udoo é un pó sporca, ma in molti altri progetti ho supportato volentieri aziende medio/piccole per progetti "visionari" che altrimenti non avrebbero potuto  sostenere.
Secondo me si é perso un pó di vista il significato del crowdfunding e si é cominciato a sfruttarlo come business model invece di incentivo alla passione di qualche matto ed é quello che sta rovinando tutto.

Il RPi3 non l'ho ancora preso e i dongle WiFi li uso pochissimo , ma con lo Zero va degnamente , anche se lo uso per protocolli IoT molto light (MQTT) quindi non posso dare un giudizio veritiero sulle prestazioni. Inoltre uso solo roba "From China with Love®" 100% quindi le prestazioni saranno un pó sotto la media...
